exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,7 @@
-const name = " Daniel  ";
+var message = "Hello, This is Farnoosh!";
+var myName = "Farnoosh";
+var myNameTrim = myName.trim();
+var myNameLenght = myName.length;
```

Comment:
This works fine 👍 
It could be even better to show the length of the trimmed name string (with no whitespace)?